### PR TITLE
Possible performance improvement - call `utils::packageVersion` only once

### PR DESCRIPTION
**Pull request by slodge**
# Summary

Small performance improvement - no longer calls `utils::packageVersion` 

# Related GitHub Issues and PRs

Fixes: [#1524](https://github.com/rstudio/gt/issues/1524)

# Checklist

- [x] I understand and agree to the [Code of Conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html).
- [x] I have listed any major changes in the [NEWS](https://github.com/rstudio/gt/blob/master/NEWS.md).
- [x] **No new tests added - none needed** I have added [`testthat`](https://github.com/r-lib/testthat) unit tests to [`tests/testthat`](https://github.com/rstudio/gt/tree/master/tests/testthat) for any new functionality.

### CODE REVIEW

File: R/zzz.R [hunk 1]
```diff
@@ -206,6 +208,14 @@ utf8_aware_sub <- NULL
...
   if (any(toset)) options(gt_default_options[toset])
 
+  # When processing text globally (outside of the `fmt_markdown()`
+  # function) we will use the 'markdown' package if it is available,
+  # otherwise the 'commonmark' package
+  gt_package_environment$md_engine <- "commonmark"
+  if (utils::packageVersion("markdown") >= "1.5") {
+    md_engine <- "markdown"
+  }
+
```
Comment (olivroy) [inline R/zzz.R:218]:
```suggestion
```

Unneeded.

**Review by olivroy — COMMENTED** [review]
Maybe this could be simplified to just using markdown? since v1.5 is required in DESCRIPTION

**Comment by slodge** [comment]
I've tried signing https://cla-assistant.io/rstudio/gt?pullRequest=1525 but maybe I've messed up some emails somewhere...

**Review by rich-iannone — APPROVED** [review]
LGTM!

**Comment by slodge** [comment]
Fixed again :) Sorry - I'd missed the latex use of `md_engine` in previous Friday afternoon commit!

**Comment by rich-iannone** [comment]
@slodge Thank you! I was just debugging this and traced it to there (*then* I noticed your commits!).

**Pull request merged**